[2023-02-24T03:42:59.173+0000] {taskinstance.py:1165} INFO - ðŸ¤ª Dependencies all met for <TaskInstance: ingest_bo_voucher_report.ingest_bo_voucher_report scheduled__2023-02-22T19:00:00+00:00 [queued]>
[2023-02-24T03:42:59.188+0000] {taskinstance.py:1165} INFO - ðŸ¤ª Dependencies all met for <TaskInstance: ingest_bo_voucher_report.ingest_bo_voucher_report scheduled__2023-02-22T19:00:00+00:00 [queued]>
[2023-02-24T03:42:59.189+0000] {taskinstance.py:1362} INFO - ðŸ¤ª 
--------------------------------------------------------------------------------
[2023-02-24T03:42:59.190+0000] {taskinstance.py:1363} INFO - ðŸ¤ª Starting attempt 1 of 4
[2023-02-24T03:42:59.191+0000] {taskinstance.py:1364} INFO - ðŸ¤ª 
--------------------------------------------------------------------------------
[2023-02-24T03:42:59.220+0000] {taskinstance.py:1383} INFO - ðŸ¤ª Executing <Task(PythonOperator): ingest_bo_voucher_report> on 2023-02-22 19:00:00+00:00
[2023-02-24T03:42:59.227+0000] {standard_task_runner.py:55} INFO - ðŸ¤ª Started process 327 to run task
[2023-02-24T03:42:59.232+0000] {standard_task_runner.py:82} INFO - ðŸ¤ª Running: ['airflow', 'tasks', 'run', 'ingest_bo_voucher_report', 'ingest_bo_voucher_report', 'scheduled__2023-02-22T19:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/ingest_bo_voucher_report.py', '--cfg-path', '/tmp/tmp_8bu7pa3']
[2023-02-24T03:42:59.234+0000] {standard_task_runner.py:83} INFO - ðŸ¤ª Job 2: Subtask ingest_bo_voucher_report
[2023-02-24T03:42:59.352+0000] {task_command.py:376} INFO - ðŸ¤ª Running <TaskInstance: ingest_bo_voucher_report.ingest_bo_voucher_report scheduled__2023-02-22T19:00:00+00:00 [running]> on host 8d8c6b853c06
[2023-02-24T03:42:59.437+0000] {taskinstance.py:1590} INFO - ðŸ¤ª Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=gp.data@galaxy.com.vn
AIRFLOW_CTX_DAG_OWNER=data_team
AIRFLOW_CTX_DAG_ID=ingest_bo_voucher_report
AIRFLOW_CTX_TASK_ID=ingest_bo_voucher_report
AIRFLOW_CTX_EXECUTION_DATE=2023-02-22T19:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-22T19:00:00+00:00
[2023-02-24T03:42:59.484+0000] {taskinstance.py:1851} ERROR - ðŸ¤ª Task failed with exception
Traceback (most recent call last):
  File "/opt/bitnami/python/lib/python3.8/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/opt/bitnami/python/lib/python3.8/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/airflow/dags/processing/ingest_bo_voucher_report.py", line 60, in ingest_bo_voucher_report
    spread = config_gg_sheet()
  File "/airflow/dags/processing/ingest_bo_voucher_report.py", line 11, in config_gg_sheet
    c = conf.get_config(path, "credentials.json")
  File "/opt/bitnami/python/lib/python3.8/site-packages/gspread_pandas/conf.py", line 88, in get_config
    raise IOError(
OSError: No Google client config found.
Please download json from https://console.developers.google.com/apis/credentials and save as /Users/admin/Desktop/GalaxyPlay/bo_ingest_voucher_report/credentials.json
[2023-02-24T03:42:59.495+0000] {taskinstance.py:1401} INFO - ðŸ¤ª Marking task as UP_FOR_RETRY. dag_id=ingest_bo_voucher_report, task_id=ingest_bo_voucher_report, execution_date=20230222T190000, start_date=20230224T034259, end_date=20230224T034259
[2023-02-24T03:42:59.506+0000] {standard_task_runner.py:100} ERROR - ðŸ¤ª Failed to execute job 2 for task ingest_bo_voucher_report (No Google client config found.
Please download json from https://console.developers.google.com/apis/credentials and save as /Users/admin/Desktop/GalaxyPlay/bo_ingest_voucher_report/credentials.json; 327)
[2023-02-24T03:42:59.526+0000] {local_task_job.py:159} INFO - ðŸ¤ª Task exited with return code 1
[2023-02-24T03:42:59.540+0000] {taskinstance.py:2623} INFO - ðŸ¤ª 0 downstream tasks scheduled from follow-on schedule check
